Cython/Debugger/Tests/TestLibCython.py

```diff
@@ -195,7 +195,7 @@ def excepthook(type, value, tb):
             end
             ''')
 
-        code = textwrap.dedent('''\
+        code = textwrap.dedent('''
```

Comment:
`../Cygdb.py:99` doesn't include a tailing line break, so I fixed it here since it seemed like the solution with the minimal change to behavior. Based on the willingness to remove dependency on `add_metaclass`, I'm going to switch.